docs/specs/shapes.md

```diff
@@ -663,3 +663,28 @@ separately, $start$ and $end$ being applied to each shape.
 * When `m` has a value of `2` (Sequential), all the shapes MUST be considered
 as following each other in render order.  $start$ and $end$ refer to the whole
 length created by concatenating each shape.
+
+
+<h3 id="trim-path">Rounded Corners</h3>
+
+{schema_string:shapes/rounded-corners/description}
+
+{schema_object:shapes/rounded-corners}
+
+<lottie-playground example="rounded_corners.json">
+    <form>
+        <input title="Roundness" type="range" min="0" value="100" max="300"/>
+    </form>
+    <json>lottie.layers[0].shapes[5]</json>
+    <script>
+        lottie.layers[0].shapes[5].s.r = data["Roundness"];
+    </script>
+</lottie-playground>
+
+Note: it does not affect:
```

Comment:
I think this won't affect paths if the control points on both of the vertices for a given edge are set? I'm not sure if that is explicit or not, was playing around with different possibilities in AE. There also seems like some edge cases where the corners get sharper?